Translators/src/artofillusion/translators/OBJImporter.java

```diff
@@ -32,23 +34,19 @@
    */
   public static Scene importFile(File f) throws Exception
   {
-    String objName = f.getName();
-    if (objName.lastIndexOf('.') > 0)
-      objName = objName.substring(0, objName.lastIndexOf('.'));
+    String objName = FileUtils.getNameNoExtension(f);
     File directory = f.getCanonicalFile().getParentFile();
 
     // Create a scene to add objects to.
 
     Scene theScene = new Scene();
     CoordinateSystem coords = new CoordinateSystem(new Vec3(0.0, 0.0, Camera.DEFAULT_DISTANCE_TO_SCREEN), new Vec3(0.0, 0.0, -1.0), Vec3.vy());
     ObjectInfo info = new ObjectInfo(new SceneCamera(), coords, "Camera 1");
-    info.addTrack(new PositionTrack(info), 0);
-    info.addTrack(new RotationTrack(info), 1);
-    theScene.addObject(info, null);
+
+    theScene.addObject(info, (UndoRecord)null);
```

Comment:
Why the Cast to `UndoRecord`?

Reply:
Just a copy-paste from ArtOfIllusion.newWindow()